src/servers/main.cc

```diff
@@ -462,9 +462,8 @@ std::vector<Option> options_
       {OPTION_TRACE_FILEPATH, "trace-file", Option::ArgStr,
        "Set the file where trace output will be saved."},
       {OPTION_TRACE_LEVEL, "trace-level", Option::ArgStr,
-       "Set the trace level. OFF to disable tracing, TIMESTAMPES to trace "
-       "timestamps, TENSORS to trace both timestamps and tensors. Default is "
-       "OFF."},
+       "Specify a trace level. OFF to disable tracing, TIMESTAMPES to "
```

Comment:
Fix spelling of TIMESTAMPS. Also you need to say something about how the flag can be specified multiple times. That is needed so that you can turn on both TIMESTAMPS and TENSORS.

Reply:
Still need to fix TIMESTAMPES -> TIMESTAMPS